core/frontend/src/components/speedtest/Graph.vue

```diff
@@ -71,10 +71,11 @@ export default Vue.extend({
     },
   },
   methods: {
-    updateSeries(series: Data) {
+    async updateSeries(series: Data) {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const chart = this.$refs.chart as any
```

Comment:
We could define a `chart` type to prevent using `any`

Reply:
So, I did try creating a d.ts but without luck, the typescript was just failing to find it at all.